frontend/src/components/ParticipantTableComponent/PrevPointTableBody.js

```diff
@@ -47,6 +47,9 @@ const PrevPointTableBody = props => {
             <TableCell aria-label="tcell" className={classes.mobileVisibility}>
               <PrevPointCopy>{participant.date_of_birth}</PrevPointCopy>
             </TableCell>
+            <TableCell aria-label="tcell">
```

Comment:
Can you add the mobile visibility class here? and also move it to left in the table, lets put it after last name

Reply:
I addressed the issue.